### DISCUSSION TITLE
In Javascript

### DISCUSSION THREAD

**Discussion by smierx** [mProjectsCode/obsidian-meta-bind-plugin#363] [Q&A] [answered]
I want to use  functions from a js file and call via dataviewjs 
how can i access the functions/interfaces

**Comment by mProjectsCode** [comment, marked as answer]
Get a reference to the plugin, it has a property called `api`. Docs for the API are [here](<https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/api/classes/obsidianapi/>).
